[2024-10-04 07:30:06,489] WARN [SocketServer listenerType=ZK_BROKER, nodeId=0] Unexpected error from /165.154.12.82; closing connection (org.apache.kafka.common.network.Selector)
org.apache.kafka.common.network.InvalidReceiveException: Invalid receive (size = 369295617 larger than 104857600)
	at org.apache.kafka.common.network.NetworkReceive.readFrom(NetworkReceive.java:105)
	at org.apache.kafka.common.network.KafkaChannel.receive(KafkaChannel.java:452)
	at org.apache.kafka.common.network.KafkaChannel.read(KafkaChannel.java:402)
	at org.apache.kafka.common.network.Selector.attemptRead(Selector.java:674)
	at org.apache.kafka.common.network.Selector.pollSelectionKeys(Selector.java:576)
	at org.apache.kafka.common.network.Selector.poll(Selector.java:481)
	at kafka.network.Processor.poll(SocketServer.scala:989)
	at kafka.network.Processor.run(SocketServer.scala:892)
	at java.base/java.lang.Thread.run(Thread.java:829)
[2024-10-04 07:30:37,575] WARN [SocketServer listenerType=ZK_BROKER, nodeId=0] Unexpected error from /165.154.12.82; closing connection (org.apache.kafka.common.network.Selector)
org.apache.kafka.common.network.InvalidReceiveException: Invalid receive (size = 1195725856 larger than 104857600)
	at org.apache.kafka.common.network.NetworkReceive.readFrom(NetworkReceive.java:105)
	at org.apache.kafka.common.network.KafkaChannel.receive(KafkaChannel.java:452)
	at org.apache.kafka.common.network.KafkaChannel.read(KafkaChannel.java:402)
	at org.apache.kafka.common.network.Selector.attemptRead(Selector.java:674)
	at org.apache.kafka.common.network.Selector.pollSelectionKeys(Selector.java:576)
	at org.apache.kafka.common.network.Selector.poll(Selector.java:481)
	at kafka.network.Processor.poll(SocketServer.scala:989)
	at kafka.network.Processor.run(SocketServer.scala:892)
	at java.base/java.lang.Thread.run(Thread.java:829)
[2024-10-04 07:30:38,006] WARN [SocketServer listenerType=ZK_BROKER, nodeId=0] Unexpected error from /165.154.12.82; closing connection (org.apache.kafka.common.network.Selector)
org.apache.kafka.common.network.InvalidReceiveException: Invalid receive (size = 218762506 larger than 104857600)
	at org.apache.kafka.common.network.NetworkReceive.readFrom(NetworkReceive.java:105)
	at org.apache.kafka.common.network.KafkaChannel.receive(KafkaChannel.java:452)
	at org.apache.kafka.common.network.KafkaChannel.read(KafkaChannel.java:402)
	at org.apache.kafka.common.network.Selector.attemptRead(Selector.java:674)
	at org.apache.kafka.common.network.Selector.pollSelectionKeys(Selector.java:576)
	at org.apache.kafka.common.network.Selector.poll(Selector.java:481)
	at kafka.network.Processor.poll(SocketServer.scala:989)
	at kafka.network.Processor.run(SocketServer.scala:892)
	at java.base/java.lang.Thread.run(Thread.java:829)
[2024-10-04 07:30:38,439] ERROR Closing socket for 74.208.112.63:9092-165.154.12.82:36318-254 because of error (kafka.network.Processor)
org.apache.kafka.common.errors.InvalidRequestException: Error parsing request header. Our best guess of the apiKey is: 0
Caused by: java.nio.BufferUnderflowException
	at java.base/java.nio.HeapByteBuffer.get(HeapByteBuffer.java:182)
	at java.base/java.nio.ByteBuffer.get(ByteBuffer.java:770)
	at org.apache.kafka.common.protocol.ByteBufferAccessor.readArray(ByteBufferAccessor.java:58)
	at org.apache.kafka.common.protocol.Readable.readString(Readable.java:44)
	at org.apache.kafka.common.message.RequestHeaderData.read(RequestHeaderData.java:122)
	at org.apache.kafka.common.message.RequestHeaderData.<init>(RequestHeaderData.java:84)
	at org.apache.kafka.common.requests.RequestHeader.parse(RequestHeader.java:95)
	at kafka.network.Processor.parseRequestHeader(SocketServer.scala:999)
	at kafka.network.Processor.$anonfun$processCompletedReceives$1(SocketServer.scala:1012)
	at java.base/java.util.LinkedHashMap$LinkedValues.forEach(LinkedHashMap.java:608)
	at kafka.network.Processor.processCompletedReceives(SocketServer.scala:1008)
	at kafka.network.Processor.run(SocketServer.scala:893)
	at java.base/java.lang.Thread.run(Thread.java:829)
[2024-10-04 07:30:38,439] ERROR Exception while processing request from 74.208.112.63:9092-165.154.12.82:36318-254 (kafka.network.Processor)
org.apache.kafka.common.errors.InvalidRequestException: Error parsing request header. Our best guess of the apiKey is: 0
Caused by: java.nio.BufferUnderflowException
	at java.base/java.nio.HeapByteBuffer.get(HeapByteBuffer.java:182)
	at java.base/java.nio.ByteBuffer.get(ByteBuffer.java:770)
	at org.apache.kafka.common.protocol.ByteBufferAccessor.readArray(ByteBufferAccessor.java:58)
	at org.apache.kafka.common.protocol.Readable.readString(Readable.java:44)
	at org.apache.kafka.common.message.RequestHeaderData.read(RequestHeaderData.java:122)
	at org.apache.kafka.common.message.RequestHeaderData.<init>(RequestHeaderData.java:84)
	at org.apache.kafka.common.requests.RequestHeader.parse(RequestHeader.java:95)
	at kafka.network.Processor.parseRequestHeader(SocketServer.scala:999)
	at kafka.network.Processor.$anonfun$processCompletedReceives$1(SocketServer.scala:1012)
	at java.base/java.util.LinkedHashMap$LinkedValues.forEach(LinkedHashMap.java:608)
	at kafka.network.Processor.processCompletedReceives(SocketServer.scala:1008)
	at kafka.network.Processor.run(SocketServer.scala:893)
	at java.base/java.lang.Thread.run(Thread.java:829)
